src/components/Collections/StyledComponents.js

```diff
@@ -50,11 +50,9 @@ export const StyledCollectionHeader = styled.span`
   input {
     width: 130px;
   }
-  ${props => props.minimized && css`
-    h3 {
-      font-style: italic;
-    }
-  `}
+  i.fa {
+    margin-right: 4px;
```

Comment:
I love the chevrons! And we can make them even better by animating them as you expand/collapse collections. The simple `transition` below will make them smoothly transition between rotated and normal:
```suggestion
    margin-right: 4px;
    transition: 0.25s ease transform;
```

Reply:
Nice catch!